config/database.yml.example

```diff
@@ -17,6 +17,9 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  host: localhost
+  username: postgres
+  password: alceconbotas2
```

Comment:
Do not upload this file to github (add it to the .gitignore), as it can generate conflicts with team members, or people who review your project. Instead I recommend using a database.yml.example file, where you put the general basic structure of that file and that people who download your project can use the .example to make their own database configuration file.

Reply:
Ok, thanks for the comments i changed the database.yml.example with a general basic configuration